frontend/tests/acceptance/pages/signupPage.js

```diff
@@ -1,19 +1,26 @@
 const { I } = inject();
 
 module.exports = {
-  url: '/signup',
+  url: "/signup",
   fields: {
-    name: '//label[contains(text(),"name")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    email: '//label[contains(text(),"email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    password: '//label[contains(text(),"password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
-    confirmPassword: '//label[contains(text(),"confirm password")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    name:
+      '//label[contains(text(),"Name")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
+    email:
+      '//label[contains(text(),"Email")]/parent::div/div/input[contains(@class, "MuiInputBase-input")]',
```

Comment:
```suggestion
      '//label[contains(text(),"%s")]/parent::div//input[contains(@class, "MuiInputBase-input")]',
```
and use `util.format()` where you need the xpath